src/app/metrics/site-metrics/site-metrics.component.scss

```diff
@@ -0,0 +1,14 @@
+.metric-card {
+    height: 6.25rem;
+    padding-left: 0;
+    padding-right: 0;
+}
```

Comment:
I have a hunch most of these scss classes can be moved to inline bootstrap classes, or at least to their child component .scss file for consistency - typically we only define scss classes as a last resort, for things we cannot control with bootstrap, like specific colours and specific width/heights. It keeps styling consistent if we ever need to change things site-wide.

Reply:
Didn't know that it should be a last resort. I usually use SCSS if there will be multiple components reusing the same CSS settings so that adjusting it once adjusts it everywhere. I've followed your advice and moved into bootstrap classes.